newIDE/app/src/GameDashboard/GameAnalyticsPanel.js

```diff
@@ -157,7 +157,6 @@ export const GameAnalyticsPanel = ({ game }: Props) => {
       width: 100,
     },
     tooltipContent: {
-      backgroundColor: gdevelopTheme.chart.tooltipBackgroundColor,
```

Comment:
Why remove this?

Reply:
As it's handled by the paper automatically now, this was redundant